ui/tests/app/admin/tab/layout.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react';
+import AdminTabsLayout from '@/app/admin/@tab/layout';
+import { usePathname } from 'next/navigation';
+
+jest.mock('next/navigation');
```

Comment:
I think this can be deleted, we are no longer using jest and Vitest does not need it.